Makefile

```diff
@@ -74,6 +75,14 @@ gen-java:
 	mkdir -p ./$(PROTO_GEN_JAVA_DIR)
 	$(foreach file,$(PROTO_FILES),$(call exec-command, $(PROTOC) --java_out=./$(PROTO_GEN_JAVA_DIR) $(file)))
 
+# Generate gRPC/Protobuf implementation for Kotlin.
+.PHONY: gen-kotlin
+gen-kotlin:
+	rm -rf ./$(PROTO_GEN_KOTLIN_DIR)
+	mkdir -p ./$(PROTO_GEN_KOTLIN_DIR)
+	$(foreach file,$(PROTO_FILES),$(call exec-command, $(PROTOC) --java_out=./$(PROTO_GEN_JAVA_DIR) --kotlin_out=./$(PROTO_GEN_KOTLIN_DIR) $(file)))
```

Comment:
Can this depend on gen-java instead of re-generating the java files in their dir?

Reply:
It could but there is a relation between the java and kotlin generated classes. The kotlin file 's refer to those from Java so it might work but I've never seen kotlin generation without the `java_out` option.

Reply:
The reason why the build failed is because it has a dependency on the newer otel image with my other PR.

Reply:
@vincentfree I understand the build failure, but still not sure about the regenerating of the java files in this rule. I would give a try without to see what happens :D

Reply:
Doesn't hurt to try